src/python/pants/backend/python/macros/python_artifact.py

```diff
@@ -1,7 +1,31 @@
 # Copyright 2014 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
-from typing import Any, Dict
+import collections.abc
+from typing import Any, Dict, List, Optional, Union
+
+from pants.base.deprecated import deprecated
+
+
+def _check_entry_points(
```

Comment:
To make it easier to work with the entry points else where, it is beneficial to have a single format to expect. This one translates the user input to a predictable format.

Reply:
Sg! Name makes this sound like it's only validation though. How about `_normalize_entry_points`?